### ISSUE TITLE
How to get the full taxa of predicted hosts

### ISSUE THREAD

**Issue by WUD2018** [KennthShang/PhaBOX#13]
Hey developer,

I noted that the Cherry gives hosts of phage sequences. However, it only lists the species names. How can I get the full taxa? Is there any taxa list? 

Thanks.

PS: there are two files (virus.csv & prokaryote.csv) in the './database/cherry' file. Which one should I use?

**Comment by KennthShang** [comment]
Hi,

Thanks for using our tools. the full taxa can be found in the file "prokaryote.csv"

You can also search for it with the ETE3 toolkit: http://etetoolkit.org/docs/latest/tutorial/tutorial_ncbitaxonomy.html

Best,
Jiayu

**Comment by WUD2018** [comment]
Thanks Jiayu,

One more suggestion here: you may note that the gtdbtk has updated its database, which has a new taxa name system. Is it possible for you to update phabox and assign the phage-host names in alignment with gtdbtk (such as v2.3 release 214)

**Comment by WUD2018** [comment]
Thanks, Jiayu,

Here is the taxa table (v2.3.0 release 214):

**Comment by KennthShang** [comment]
Hi there,

I am sorry, but it seems the provided taxa table cannot be used to convert RefSeq into GTDB. I found that many of the sequences in the RefSeq taxa cannot find their corresponding taxa in the provided file (missing many accession maps).

However, I suddenly found that my provided scripts in the GTDB folder had some problems in the previous release. I have fixed the problems with a readme file. Hope it can help you to convert the RefSeq into the wanted GTDB. The GTDB taxa are also downloaded from the official website.

Best,
Jiayu
